wormhole-connect/src/hooks/useComputeDestinationTokens.ts

```diff
@@ -54,8 +55,15 @@ const useComputeDestinationTokens = (props: Props): ReturnProps => {
       // Done fetching and setting all supported tokens
       setIsFetching(false);
 
-      if (destChain && supported.length === 1) {
-        if (!canceled) {
+      if (active && destChain && destToken !== '') {
+        // check if the pre selected destToken is supported
+        const isTokenSupported = supported.some((t) => t.key == destToken);
+        if (!isTokenSupported) {
+          // if not, clear the destToken
+          dispatch(setDestToken(''));
```

Comment:
Why is this necessary? Can't we just let the UI show no quotes if there are no routes?

Reply:
you're right, removed